site/content/docs/05-layouts.md

```diff
@@ -83,4 +83,6 @@ Layout components receive a `segment` property which is useful for things like s
 +		href="settings/notifications"
 +	>Notifications</a>
 </div>
-```
\ No newline at end of file
+```
+
+If in some cases you would like to disable layout inheritance - you can place a file called `_reset.svelte` in the folder. In this case only root layout will be applied.
```

Comment:
This isn't quite clear to me. Is `_reset.svelte` an empty file? Do you put a new layout in it or if you want additional layout do you place a `layout.svelte` alongside it?